helmfile/values/networkpolicies/service/harbor.yaml.gotmpl

```diff
@@ -0,0 +1,382 @@
+{{- define "old-style.rule.gen" }}{{ if and .ips .ports }}
+{{- $peers := list }}{{ range .ips }}{{ $peers = dict "cidr" . | append $peers }}{{ end }}
+{{- $ports := list }}{{ range .ports }}{{ $ports = dict "tcp" . | append $ports }}{{ end }}
+{{- dict "peers" $peers "ports" $ports | toYaml | nindent 4 | trim -}}
+{{- end }}{{ end -}}
+
+rules:
+
+  egress-rule-harbor-core:
+    peers:
+      - podSelectorLabels:
+          component: core
+    ports:
+      - tcp: 8080
+
+  egress-rule-harbor-database:
+    peers:
+      - podSelectorLabels:
+          component: database
+    ports:
+      - tcp: 5432
+
+  egress-rule-harbor-redis:
+    peers:
+      - podSelectorLabels:
+          component: redis
+    ports:
+      - tcp: 6379
+
+  egress-rule-harbor-registry:
+    peers:
+      - podSelectorLabels:
+          component: registry
+    ports:
+      - tcp: 5000
+      - tcp: 8080
+
+  egress-rule-harbor-trivy:
+    peers:
+      - podSelectorLabels:
+          component: trivy
+    ports:
+      - tcp: 8080
+
+  egress-rule-harbor-portal:
+    peers:
+      - podSelectorLabels:
+          component: portal
+    ports:
+      - tcp: 8080
+
+  egress-rule-harbor-notary-server:
+    peers:
+      - podSelectorLabels:
+          component: notary-server
+    ports:
+      - tcp: 4443
+
+  egress-rule-harbor-jobservice:
+    peers:
+      - podSelectorLabels:
+          component: jobservice
+    ports:
+      - tcp: 8080
+
+  {{- with .Values.networkPolicies.harbor.registries }}
+  {{- if and .ips .ports }}
+  egress-rule-harbor-registries:
+    {{ template "old-style.rule.gen" . }}
+  {{- end }}{{ end }}
+
+  {{- with .Values.networkPolicies.harbor.jobservice }}
+  {{- if and .ips .ports }}
+  egress-rule-harbor-external-jobservice:
+    {{ template "old-style.rule.gen" . }}
+  {{- end }}{{ end }}
+
+  egress-rule-harbor-external-redis:
+    peers:
+    {{- range .Values.networkPolicies.harbor.redis.external }}
+      {{- if .egressNSMatchLabels }}
+    - namespaceSelector:
+        matchLabels:
+          {{- range $key, $value := .egressNSMatchLabels }}
+          {{ $key | quote }}: {{ $value | quote }}
+          {{- end }}
```

Comment:
Maybe we should change how we configure these :thinking:

Reply:
Yes I think so to. Will see if i do it now or create an issue for it.

Reply:
I have changed how we configure these. Will double check it tomorrow, but it worked for postgres.